tests/suites/test_suite_psa_crypto.function

```diff
@@ -508,76 +511,95 @@ void import_export( data_t *data,
         ASSERT_ALLOC( reexported, export_size );
     PSA_ASSERT( psa_crypto_init( ) );
 
-    psa_set_key_usage_flags( &attributes, usage_arg );
-    psa_set_key_algorithm( &attributes, alg );
-    psa_set_key_type( &attributes, type );
+#if defined(PSA_CRYPTO_DRIVER_TEST)
+#define MAX_TEST_DRIVER_LOCATIONS 2
+    uint32_t location[MAX_TEST_DRIVER_LOCATIONS] = {PSA_KEY_LOCATION_LOCAL_STORAGE , PSA_CRYPTO_TEST_DRIVER_LOCATION};
+    for( int i = 0 ; i < MAX_TEST_DRIVER_LOCATIONS ; i++ )
+    {
```

Comment:
This was requested as one of the comments earlier to avoid duplication of the tests. It only enables looping when test driver support is enabled of course.

So maybe we all need a consensus on this one !

Reply:
We've talked about this and we'll go for duplicating test cases for now. I filed https://github.com/ARMmbed/mbedtls/issues/4922 to design a better system.